tuxemon/states/idle/color_state.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from typing import Optional
+
+from tuxemon import prepare
+from tuxemon.graphics import string_to_colorlike
+from tuxemon.menu.menu import PygameMenuState
+from tuxemon.menu.theme import get_theme
+from tuxemon.platform.events import PlayerInput
+
+
+class ColorState(PygameMenuState):
+    """
+    Menu for the change of background: color
+    It acts as opposite of transition, it's not limited temporarily.
```

Comment:
What do you mean by "it's not limited temporarily"?

Reply:
OK, I see it was already this way before you just copy/pasted it... Hmm... Please if you have any idea about what it could mean, feel free to update the documentation because this sentence doesn't make any sense imo.

Reply:
because transitions (eg flash, lasts only seconds and then disappear), and yeah, I copy paste, but I was the one who wrote that in the first place

Reply:
I hope it makes sense now